src/function-builder.ts

```diff
@@ -139,12 +139,12 @@ export function region(
 /**
  * Configure runtime options for the function.
  * @param runtimeOptions Object with three optional fields:
- * 1. failurePolicy: failure policy policy of the function, boolean `true` is
- *    equivalent to providing an empty policy.
- * 2. memory: amount of memory to allocate to the function, possible values
- *    are: '128MB', '256MB', '512MB', '1GB', and '2GB'.
- * 3. timeoutSeconds: timeout for the function in seconds, possible values are
- *    0 to 540.
+ * 1. failurePolicy: failure policy of the function, with boolean true being
```

Comment:
I may have removed the backticks in my review comment but let's add them back in around `true`

Reply:
668c02fe4b9fbff28136c79e36959c1df4d23e33

Reply:
668c02fe4b9fbff28136c79e36959c1df4d23e33